.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+node_modules/
```

Comment:
- [ ] Thanks for leaving a comment while requesting a review about your concerns related to `.gitignore`. You setup everything right, but it looks like you pushed `test.md` into the repo before listing it in `.gitignore`. I recommend you to delete `test.md` file, push changes and add it again, and ignoring should work in this case.